### ISSUE TITLE
can't dismountfromowner

### ISSUE THREAD

**Issue by xkhhv** [BlockHorizons/BlockPets#151]
when i sit my pet on my head and jump it doesn't dismount why?

**Comment by Muqsit** [comment]
Yeah, there's no efficient way of dismounting when the pet rides you.

**Comment by xkhhv** [comment]
and what about when the pet level up the size and the health of it doesn't change..

**Comment by FreeGamingHere** [comment]
Dismounting kickes the players for "Flying is not enabled on this server" due to the airTicks being for than 100
